test/e2e/serial/README.md

```diff
@@ -34,5 +34,7 @@ they found it before they run.
   failed unexpectedly on standard output, alongside (not replacing) the logging of the said events.
 - `E2E_NROP_TARGET_NODE` (accepts string, e.g. `node-0.my-cluster.io`) instructs the suite to always pick the
   node matching the given name when a random node is needed.
-- `E2E_NROP_TEST_COOLDOWN` (accepts string expressing time unit, e.g. `30s`) instructs the suite to wait for
-  the specified amount of time after each spec, to give tome to the cluster to settle up.
+- `E2E_NROP_TEST_COOLDOWN` (accepts string expressing time unit, e.g. `30s`) instructs the suite to wait
+  for the specified amount of time after each spec, to give time to the cluster to settle up.
```

Comment:
there are cooldowns within the specs, maybe we can adapt using this value for them as well instead of only "after each spec"

Reply:
I'd prefer to keep them separated, because the cooldown should be gone in the long run (the sooner the better, but we need to catch and fix any regression caused by this band-aid)

Reply:
sorry, I misread. These cooldowns are yet another story and we need to tackle them on a case-by-case

Reply:
And yes, these cooldowns are really ugly. I can say that because I added the biggest share of them.

Reply:
it's fine, then we can handle these later